<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalEstimationDialog</class>
 <widget class="QDialog" name="NormalEstimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="buttonPCA">
     <property name="text">
      <string>PCA Estimate Normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="supportSizeLabel">
        <property name="text">
         <string>Support Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_pca_neighbors">
        <property name="suffix">
         <string>Nearest Neighbors</string>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="buttonJet">
     <property name="text">
      <string>Jet Estimate Normals</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="supportSizeLabel_2">
        <property name="text">
         <string>Support Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_jet_neighbors">
        <property name="suffix">
         <string>Nearest Neighbors</string>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="buttonVCM">
     <property name="text">
      <string>VCM Estimate Normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="offsetRadiusLabel">
        <property name="text">
         <string>Offset Radius</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="m_offset_radius">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="m_convolution_radius"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="buttonRadius">
        <property name="text">
         <string>Convolution Radius</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="buttonNeighbors">
        <property name="text">
         <string>Convolution Neighbors</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="m_convolution_neighbors">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="buttonOrient">
     <property name="text">
      <string>Orient Estimated Normals</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="neighborhoodSizeLabel">
        <property name="text">
         <string>Neighborhood Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_orient_neighbors">
        <property name="suffix">
         <string>Nearest Neighbors</string>
        </property>
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="value">
         <number>18</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonJet</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPCA</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonVCM</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOrient</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRadius</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_convolution_radius</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNeighbors</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_convolution_neighbors</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NormalEstimationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NormalEstimationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
